AdSec/Python/ApiToAds.py

```diff
@@ -0,0 +1,45 @@
+import oasys.adsec
+from Oasys.AdSec import ISection, ILoad
+from Oasys.AdSec.DesignCode import EN1992
+from Oasys.Collections import IList
+from Oasys.AdSec.StandardMaterials import Concrete
+from Oasys.Profiles import ICircleProfile
+from Oasys.AdSec.IO.Serialization import JsonConverter
+from UnitsNet import Length, Force
+from UnitsNet.Units import LengthUnit, ForceUnit
+from Oasys.Units import Moment,MomentUnit
+
+# This example shows how to convert an API object into .ads file
+#
+# You might like to run the 'version.py' example first, just to check
+# that the API is installed correctly.
+
+def api_to_ads():
+    # Create API section
+    circle_profile = ICircleProfile.Create(Length(float(1000), LengthUnit.Millimeter))
```

Comment:
I appreciate that this is a sample script rather than one that we expect the customers to use as is, but can it be updated to accept an existing profile as an argument and thus make it a function that can be imported into another script?